### PR TITLE
Optimize-the-delayAfterRunSeconds-test-case-run-time

### PR DESCRIPTION
**Pull request by ChunxiAlexLuo**
Signed-off-by: Chunxi Luo <chuluo@redhat.com>

https://github.com/open-cluster-management-io/governance-policy-propagator/pull/38#issuecomment-1196813082

### CODE REVIEW

**Comment by ChunxiAlexLuo** [comment]
before
Screen Shot 2022-07-28 at 12 25 52 AM

after
Screen Shot 2022-07-28 at 12 23 11 AM

**Comment by ChunxiAlexLuo** [comment]
cc/ @mprahl

File: test/e2e/case5_policy_automation_test.go [hunk 1]
```diff
@@ -595,6 +596,22 @@ var _ = Describe("Test policy automation", func() {
...
 
+			By("Patching automationStartTime to an earlier time and let delayAfterRunSeconds expire immediately")
+			policyAutomation, err = clientHubDynamic.Resource(gvrPolicyAutomation).Namespace(testNamespace).Get(
+				context.TODO(), "create-service-now-ticket", metav1.GetOptions{},
+			)
+			Expect(err).To(BeNil())
+			status := policyAutomation.Object["status"].(map[string]interface{})
+			clustersWithEvent := status["clustersWithEvent"].(map[string]interface{})
+			for _, ClusterEvent := range clustersWithEvent {
+				updateStartTIme := time.Now().UTC().Add(-241 * time.Second).Format(time.RFC3339)
```
Comment (mprahl) [inline test/e2e/case5_policy_automation_test.go:607]:
```suggestion
				updateStartTime := time.Now().UTC().Add(-241 * time.Second).Format(time.RFC3339)
```

**Review by mprahl — APPROVED** [review]

**Review by mprahl — APPROVED** [review]

**Pull request merged**